support/latest/slt.md

```diff
@@ -2,26 +2,17 @@
+You can open a new support case or view your historical support cases at any time via [Community](https://community.hyland.com/){:target="_blank"} on the Open Support Cases page. Below are the target response times for a support case based assigned severity level.
 
-Under Developer Support, the initial response target for all cases is one business day regardless of severity. At this time, there is no target offered for case resolution.
+Support center phone numbers are available via the [Contact Support](https://community.hyland.com/customer-portal/wiki/hyland-support/technical-support-phone-numbers){:target="_blank"} link on Community.
 
-All support center phone numbers are available via the **Contact Support** link on the home page or you can search for **Support Numbers** within the [Support Portal](https://support.alfresco.com){:target="_blank"}.
+Service Level Target Response Times
 
-Initial Service Level Target Response Times
-
-|Severity|Short Title|Premier/Partner|Enterprise|Starter/Business|
-|--------|-----------|---------------|----------|----------------|
-|One (1)|Production System Down|1 hour|2 hours|4 Business hours|
-|Two (2)|Production/Development - Major Impact|2 hours|4 Business hours|1 Business day|
-|Three (3)|Production/Development - Minor Impact|6 hours|1 Business day|2 Business days|
-|Four (4)|Question/How-to/Enhancement|1 Business day|2 Business days|3 Business days|
-
-Subsequent Service Level Target Response Times
-
-|Severity|Short Title|Premier/Partner|Enterprise|Starter/Business|
-|--------|-----------|---------------|----------|----------------|
-|One (1)|Production System Down|Constant Phone Bridge|Constant Phone Bridge|Business Hourly|
-|Two (2)|Production/Development - Major Impact|Business Daily|Business Daily|Every two business days|
-|Three (3)|Production/Development - Minor Impact|Every two business days|Every two business days|Every three business days|
-|Four (4)|Question/How-to/Enhancement|Every four business days|Every four business days|Every five business days|
+| Severity | Hyland Response |
+| -------- | --------------- |
+| Level 1 | Upon receiving notification from a customer, a Hyland support Manager is immediately notified.  Within thirty (30) minutes, the Manager will notify a member of Support Senior management.<br><br>If there is no resolution within two (2) hours of a customer’s notice, Hyland will place the customer on the High Visibility Ticker (HVT).<br><br>If there is no resolution within four (4) hours of the customer’s notice or by the end of business of that day, Hyland will designate the Error as Code Blue.  Designation as Code Blue means a resolution team is immediately formed for the Error and the resolution team provides continuous updates to the C-Level Executives and Vice Presidents of Hyland.<br><br>To provide a resolution, Hyland will match the customer’s effort, up to and including 24 hours days, 7 days a week, through holidays and weekends until there is a resolution. |
```

Comment:
the numbering here goes onto the line underneath also like I described above. cheers.

Reply:
I don' think we can fix this